Cython/Compiler/PyrexTypes.py

```diff
@@ -168,6 +169,7 @@ class PyrexType(BaseType):
     #  Base class for all Cython types
     #
     #  is_pyobject           boolean     Is a Python object type
+    #  is_hpy                boolean     Is an HPy handle type
```

Comment:
Do we need this distinction?

Reply:
No, that's a left-over. I just overlooked it.